packages/react-server/core/ReactServerAgent/Request.js

```diff
@@ -101,6 +101,20 @@ Request.prototype.type = function (type) {
 	return this;
 }
 
+Request.prototype.toJSON = function(){
+	return {
+		aborted: this.aborted,
+		agent: this._agent,
+		headers: this._headers,
+		method: this._method,
+		postParams: this._postParams,
+		queryParams: this._queryParams,
+		timeout: this._timeout,
+		type: this._type,
+		urlPath: this._urlPath,
```

Comment:
This LGTM.

@roblg - Do I remember you adding some special handling for this outside of React Server core?

Any conflict here?